pkg/config/server/server.go

```diff
@@ -598,10 +598,11 @@ type ServerConfig struct {
 }
 
 type PayloadStoreConfig struct {
-	EnablePayloadDualWrites        bool          `mapstructure:"enablePayloadDualWrites" json:"enablePayloadDualWrites,omitempty" default:"false"`
-	WALPollLimit                   int           `mapstructure:"walPollLimit" json:"walPollLimit,omitempty" default:"1000"`
-	WALProcessInterval             time.Duration `mapstructure:"walProcessInterval" json:"walProcessInterval,omitempty" default:"15s"`
-	ExternalCutoverProcessInterval time.Duration `mapstructure:"externalCutoverProcessInterval" json:"externalCutoverProcessInterval,omitempty" default:"15s"`
+	EnablePayloadDualWrites          bool          `mapstructure:"enablePayloadDualWrites" json:"enablePayloadDualWrites,omitempty" default:"true"`
+	EnableTaskEventPayloadDualWrites bool          `mapstructure:"enableTaskEventPayloadDualWrites" json:"enableTaskEventPayloadDualWrites,omitempty" default:"true"`
```

Comment:
@abelanger5 wdyt of defaulting these to true? I think if they're false, people who are self-hosting will have failures if they deploy + try to replay, etc.

Reply:
yep, sounds good